docs/csharp/language-reference/keywords/nameof.md

```diff
@@ -142,8 +142,21 @@ NameOf(o.Equals) -> ' result "Equals".  Warning: "Access of static member of ins
   
  Because the argument needs to be an expression syntactically, there are many things disallowed that are not useful to list.  The following are worth mentioning that produce errors: predefined types (for example, `int` or `void`), nullable types (`Point?`), array types (`Customer[,]`), pointer types (`Buffer*`), qualified alias (`A::B`), and unbound generic types (`Dictionary<,>`), preprocessing symbols (`DEBUG`), and labels (`loop:`).  
   
- If you need to get the fully-qualified name, you can use the `typeof` expression along with `nameof`.  
+ If you need to get the fully-qualified name, you can use the `typeof` expression along with `nameof`.  For example:
+```csharp  
+void f(int i) {  
```

Comment:
Could you wrap this in the C class as you did in the next example so that it can be compilable?